### DISCUSSION TITLE
Problems

### DISCUSSION THREAD

**Discussion by rajkly** [TenderOwl/Frog#24] [General]
Hello, friends!

I can't install de application, can anyone help me?

**Comment by amka** [comment]
Hey @rajkly! Could you provide any additional info?

**Comment by rajkly** [comment]
I tried to install with the builder, but the command is not working. And i tried to install as flatkpak in terminal, but is the same, and found only this:
1) app/org.gnome.frogr/x86_64/stable
2) app/com.endlessnetwork.frogsquash/x86_64/stable
